pkg/apiserver/config/config.go

```diff
@@ -79,6 +85,11 @@ import (
 // mysql-host is missing in command line flags, all other mysql command line flags
 // will be ignored.
 
+var (
+	once        sync.Once
+	cfgChangeCh = make(chan Config)
```

Comment:
Why not create cfgChangeCh as a local variable, I see nowhere else use it.

Reply:
Now the `viper` initialization of the `TryLoadFromDisk` function is only executed once, and there are some problems with defining it as a local variable. I am going to optimize the cofnig package in another PR

Reply:
The once in TryLoadFromDisk is added by me. The initialization of viper only needs to be executed once, especially the initialization of OnConfigChange. If the variable is defined in once of TryLoadFromDisk, according to the current function signature, it will not be accessible outside of once. If placed outside of once, multiple calls to TryLoadFromDisk will initialize multiple channels. 

I think the initialization of the configuration and the opening of the watch config channel should be separated from TryLoadFromDisk and exposed as a separate api. Or config is a singleton

I will optimize the config package in this pr